### ISSUE TITLE
Set the same MTU for the NSC internal interface as the base interface has

### ISSUE THREAD

**Issue by ljkiraly** [networkservicemesh/sdk-vpp#514]
## Description

In case of vlan remote mechanism the MTU of the interface injected in NSC Pod is not adjusted.

## Possible solution

Fetch the base interface MTU and include it into the connection.Context.

**Comment by edwarnicke** [comment]
@ljkiraly Last I checked... the MTU on the interface in the NSC was set as the smallest MTU along the path, where the MTU for tunneled interfaces was calculated by subtracting overhead from the underlying interface.  What are you seeing that differs from that?

**Comment by ljkiraly** [comment]
@edwarnicke Yes, for vxlan mechanism the MTU is set correctly. I missed to adjust the MTU in case of remote vlan mechanism.

**Comment by ljkiraly** [comment]
Based on kernel [documentation](https://www.kernel.org/doc/html/v5.8/networking/netdevices.html#mtu) the MTU does not include link layer header overhead.

**Comment by edwarnicke** [comment]
@ljkiraly You are correct.
